Add games/runelite

Limero opened the pull request:
Open source RuneScape game client [RuneLite](https://runelite.net/)

[Sent to ports@ 2023-02-09](https://marc.info/?l=openbsd-ports&m=167597225623598)

rfht commented:
hi, I'm interested in following up on this. @Limero, could you rebase this (`FINISHED` file)?

Limero commented:
@rfht Thanks! I've rebased, updated the client version (not really needed as it auto updates anyways), changed to using `SITES` and bumped GTK version. Tested and confirmed working on OpenBSD 7.4
